utils.py

```diff
@@ -17,6 +21,26 @@ def perform_notebook_conversion(notebook_json):
     return nb_html
 
 
+def perform_rmd_conversion(stream):
+    incoming_data = FileStorage(stream)
+    
+    # The rmarkdown converter unfortunately only works with file paths.
```

Comment:
A bit unfortunate we need to create tmp files for the R Markdown conversion, but seems to work okay